matrix-sdk-android/src/main/java/org/matrix/android/sdk/internal/crypto/store/db/migration/MigrateCryptoTo019.kt

```diff
@@ -30,7 +30,7 @@ import org.matrix.android.sdk.internal.util.database.RealmMigrator
  * mark existing keys as safe.
  * This migration can take long depending on the account
  */
-internal class MigrateCryptoTo019(realm: DynamicRealm) : RealmMigrator(realm, 18) {
+internal class MigrateCryptoTo019(realm: DynamicRealm) : RealmMigrator(realm, 19) {
```

Comment:
Can this cause any issues?

Reply:
Hopefully no. The target version is only used in `RealmMigrator` for log purpose.